armory/utils/metrics.py

```diff
@@ -11,17 +11,94 @@
 from contextlib import contextmanager
 import io
 from collections import defaultdict, Counter
+import os
 from typing import Dict, List
 
 import cProfile
 import pstats
 from scipy import stats
 
+from armory import paths
 from armory.data.adversarial.apricot_metadata import (
     ADV_PATCH_MAGIC_NUMBER_LABEL_ID,
     APRICOT_PATCHES,
 )
 from armory.logs import log
+from armory.utils.external_repo import ExternalPipInstalledImport
+
+# TODO: update label_mapping with total counts after measurement PR
+_ENTAILMENT_MODEL = None
+
+
+class Entailment:
+    # TEST Reference (from Mike):
+    # For reference, on the first 100 benign predictions, I got 94 entailment and 6 contradictions. The Eval 5 target phrases should produce 100 contradictions.
```

Comment:
could you add a description of what this is measuring?

Reply:
docstring added